manager/src/main/java/com/redhat/service/bridge/manager/actions/sendtobridge/SendToBridgeActionTransformer.java

```diff
@@ -24,7 +32,13 @@ public BaseAction transform(BaseAction action, String bridgeId, String customerI
         Bridge destinationBridge = bridgesService.getAvailableBridge(destinationBridgeId, customerId);
 
         Map<String, String> parameters = new HashMap<>();
-        parameters.put(WebhookAction.ENDPOINT_PARAM, destinationBridge.getEndpoint());
+
+        try {
+            parameters.put(WebhookAction.ENDPOINT_PARAM, getBridgeWebhookUrl(destinationBridge.getEndpoint()));
+        } catch (MalformedURLException e) {
+            LOG.error("Error when translating bridge endpoint to webhook URL", e);
```

Comment:
@kostola Does this result in duplicate stacktrace printed in the log? We have `error` level logging in the `EventBridgeManagerExceptionMapper` which should print all failures.

Reply:
You're right. I removed it :+1: